inventory/host_vars/example.org/config.yml

```diff
@@ -1,3 +1,13 @@
 ---
 database_name: db_name
 rails_environment: env
+
+sys_admins:
+  - name: sys_admin_username
+    ssh_key: "../pub_keys/sys_admin_username.pub"
+    state: present
+
+developers:
+  - name: developer_username
+    ssh_key: "../pub_keys/developer_username.pub"
+    state: present
```

Comment:
can we move this to documentation? It leads you to think something different than what's intended.

Reply:
Sure, will do in https://github.com/coopdevs/timeoverflow-provisioning/issues/22